packages/manager/modules/pci/src/components/auto-generate-name/auto-generate-name.js

```diff
@@ -0,0 +1,22 @@
+const maxRandomNumber = 9999;
+
+function getDate() {
+  const date = new Date();
+  return `${date.getDate()}${date.getMonth() + 1}`;
+}
+
+function getRandomNumber() {
+  return (
+    (Math.floor(Math.random() * maxRandomNumber) *
+      new Date().getMilliseconds()) %
+    maxRandomNumber
+  );
+}
+
+export const getAutoGeneratedName = (prefix = '') => {
+  return `${prefix}-${getDate()}-${getRandomNumber()}`;
```

Comment:
So if no prefix is specified it will start with `-`, seems strange no ?

Reply:
Done https://github.com/ovh/manager/commit/684a600dc5c9158052884ba1e5fb6499e17368e2